### PR TITLE
theme: make small ui fixes

### PR DESCRIPTION
**Pull request by Dinika**
## Description
Changes made in detailed literature view are:
i. Extra spaces between authors and e-print fields is removed
ii. Scroll bars are removed when they are not needed
iii. Leading dash in front of date is removed when no authors or collaborations are present
iv. Position the search glyphicon in the center of the button

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Dinika** [comment]
##  Extra spaces between authors and e-print fields is removed
1 so much space record_journal
Before
2 extra space removed
After the fix

## Scroll bars are removed when they are not needed
5 scroll bars even when not needed
Before
6 no scroll bars when not needed
After the fix

##  Leading dash in front of date is removed when no authors or collaborations are present
3 date with dash
Before
4 without dash date
After the fix

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl [hunk 1]
```diff
@@ -39,17 +39,23 @@
           <h1 class="record-detailed-title">{{ render_record_title(record) }}</h1>
           <div id="record-authors">
             {{ render_record_authors(record, is_brief=false, number_of_displayed_authors=25) }}
-            <span id="record-date">- {{ record.get('earliest_date')|format_date }}</span>
+            <span id="record-date">
+            {% if record.get('authors') or record.get('collaborations') %}
+            - {{ record.get('earliest_date')|format_date }}
+            {% else %}
+              {{ record.get('earliest_date')|format_date }}
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl]: Why is the `if-else` needed if in the end the same code gets executed?

Comment (jacquerie) [reply inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl]: (It's the dash!)

**Comment by jmartinm** [comment]
Merged in https://github.com/inspirehep/inspire-next/pull/2882

**Pull request closed without merging**